modules/ROOT/pages/errors/index.adoc

```diff
@@ -1,21 +1,91 @@
-:description: The Neo4j error codes for Neo4j version {neo4j-version}.
-
-[[error-codes]]
+:description: The Neo4j error codes for Neo4j version.
+[[neo4j-errors]]
 = Server errors
 
-Error codes are Neo4j status codes returned by the server when the execution of a query fails.
-They always have the severity level `ERROR`.
-Errors are grouped based on the type of the error code:
+[role=label--version-5.25]
+[[gqlstatus-error-object]]
+== GQL-status error object
+
+[[gqlstatus-standard-defined-codes]]
+=== Standard-defined GQLSTATUS codes
 
-Client errors::
-    These errors are caused by the client and are usually related to the request itself.
-    Client errors have the prefix `Neo.ClientError`.
+.Standard-defined GQLSTATUS error codes
```

Comment:
I wonder if we need to introduce standard-defined and Neo4j-defined GQLSTATUS codes separately from user's point of view. What is difference (for users) if they get standard-defined code not the neo4j-defined? I suppose if they get an error and see a code, they should go directly to the _List of all GQLSTATUS server error codes_ for more information.

Reply:
I don't know whether it is a good or a bad idea. I just started from them but then decided to move to the neo4j-specific ones, in order to create a template for both of us to work simultaneously. So you don't need to review them.